packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/enhanced/cross_document_intelligence.py

```
@@ -14,23 +14,32 @@
-import math
 import warnings
-import networkx as nx
+import numpy as np
+import asyncio
 from collections import defaultdict, Counter
 from dataclasses import dataclass, field
-from enum import Enum
-from typing import Any, Dict, List, Optional, Set, Tuple, Union
 from datetime import datetime
+from enum import Enum
+from typing import Any, Optional, List, Dict, Tuple, Union, TYPE_CHECKING
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)